constants/megaMenuLinks.ts

```diff
@@ -172,14 +172,14 @@ const GAMES_AND_DEMOS: MegaDropdownMenuGroup = {
     },
     {
       label: 'Bonus Level: Make your own puzzles',
-      url: `${baseUrl}${pathGamesAndDemos}/bonus-level-make-your-own-puzzles`,
+      url: `${baseUrl}${pathGamesAndDemos}/bonus-level-sandbox#bonus-148-0`,
       segment: {
         cta: 'bonus-level-make-your-own-puzzles', location: sectionGamesAndDemos
       }
     },
     {
       label: 'Level 5: Proving the Uniqueness of Quantum Variables',
-      url: `${baseUrl}${pathGamesAndDemos}/level-5-proving-the-uniqueness-of-quantum-variables`,
+      url: `${baseUrl}${pathGamesAndDemos}/bonus-level-sandbox#bonus-153-0`,
```

Comment:
do we need these two sections? they are part of the same page as the item above (Bonus Level: Sandbox)?
in addition removing them will make it consistent with platypus:

**qiskit.org**
![image](https://user-images.githubusercontent.com/13156555/168179886-c5e341fb-aa15-4f1e-8a0a-eae5003f2a5b.png)

**platypus**
![image](https://user-images.githubusercontent.com/13156555/168180012-b6a592c5-28c3-46b4-b659-76bad05fa0f4.png)

i think these two should be consistent (which i know should come once is https://github.com/Qiskit/web-components/issues/74 & https://github.com/Qiskit/platypus/issues/955 are addressed)

Reply:
Good call... will fix now

Reply:
@vabarbosa - should be fixed and consistent w/ Textbook now...
![Screen Shot 2022-05-12 at 4 13 29 PM](https://user-images.githubusercontent.com/6276074/168182382-2fb16f77-cde5-4e35-acec-888c1da1adb6.png)